packages/caliper-ethereum/package.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "caliper-ethereum",
+  "version": "0.1.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/hyperledger/caliper"
+  },
+  "scripts": {
+    "test": "npm run lint",
+    "lint": "npx eslint .",
+    "nyc": "nyc mocha --recursive -t 10000"
+  },
+  "engines": {
+    "node": ">=8.10.0",
+    "npm": ">=5.6.0"
+  },
+  "engine-strict": true,
+  "engineStrict": true,
+  "files": [
+    "*",
+    "src/comm/template/report.html"
+  ],
+  "dependencies": {
+    "caliper-core" : "0.1.0",
+    "web3": "^1.2.0"
```

Comment:
Web3 should be put under devDeps. It will be installed during binding, similarly to the other adapters. So you need to modify the following file also: 
https://github.com/hyperledger/caliper/blob/6ddd6003da8772e9065cf5f179b177e8b47705d9/packages/caliper-cli/lib/bind/config.yaml#L29

```yaml
sut:
  ethereum:
    1.0.0:
      packages: [ 'web3@1.0.0' ]
```
There could be more 1.X.0 entries of course.

Reply:
I added the latest version, I don't know if it can be usufull to add all the 1.X ones

Reply:
@russanto this is awesome! Please do a squash commit so there is only one commit being merged.